src/app/dataset-view/dataset-view.interface.ts

```diff
@@ -6,3 +15,21 @@ export enum DatasetViewTypeEnum {
     discussions = "discussions",
     history = "history",
 }
+
+export interface DatasetViewContentInterface {
+    isTableHeader: boolean;
+    displayedColumns?: any[];
+    datasetDataSource?: SearchHistoryInterface[];
+    datasetOverviewSource?: SearchHistoryInterface[];
+    datasetMetadataSource?: SearchOverviewDatasetsInterface[];
+    datasetHistorySource?: MetadataBlockFragment[];
```

Comment:
Most of the fields seems to reflect parameters for a single table instance, but what are these?

Is this type supposed to represent the state of a single table or the entire dataset view? Right now it seems to be somewhere in between.